keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt

```diff
@@ -61,6 +61,16 @@ internal class ApplicationControllerTests : JUnit5Minutests {
     const val user = "keel@keel.io"
   }
 
+  val payloadWithLongComment =
+    """{
+        |  "version": "master-h22.0e0310f",
+        |  "reference": "my-artifact",
+        |  "targetEnvironment": "testing",
+        |  "comment": "Bacon ipsum dolor amet turducken prosciutto shoulder ground round hamburger, flank frankfurter rump ham hock sirloin leberkas meatloaf shankle landjaeger pig.  Shoulder shankle doner ball tip burgdoggen kevin alcatra bresaola.  Leberkas alcatra cow, sausage picanha chislic tongue hamburger turkey tail chicken flank.  Chislic tri-tip ribeye meatloaf swine biltong rump.  Bacon salami ball tip, kevin turkey cow short loin cupim drumstick landjaeger rump meatball flank.",
+  "Burgdoggen leberkas pork loin salami ribeye pig, ham meatloaf sirloin beef prosciutto chuck cupim pork swine.  Bacon cupim doner pig drumstick prosciutto pork loin picanha beef ribs pork.  Pig beef strip steak buffalo short ribs swine jowl short loin alcatra.  Frankfurter spare ribs cupim leberkas.  Ham hock cow venison boudin ham tenderloin chuck chislic porchetta kevin leberkas.  Tri-tip buffalo meatball leberkas tail.",
```

Comment:
I think this might break the test because of the comma before. You probably only need the first paragraph?

Reply:
shortened the comment 👍